packages/payload/src/utilities/logError.ts

```diff
@@ -0,0 +1,25 @@
+import type pino from 'pino'
+
+import type { Payload } from '../types/index.js'
+
+export const logError = ({ err, payload }: { err: unknown; payload: Payload }): void => {
+  let level: pino.Level = 'error'
```

Comment:
The type should be `pino.Level | false`, if we were in strict mode L15 would error since it doesn't match the `loggingLevels` type from the config.